airflow/providers/amazon/aws/operators/sagemaker.py

```diff
@@ -222,6 +230,7 @@ def __init__(
         self.max_attempts = max_attempts or 60
         self.max_ingestion_time = max_ingestion_time
         self.deferrable = deferrable
+        self.serialized_job: dict[Any, Any] | None = None
```

Comment:
```suggestion
        self.serialized_job: dict | None = None
```
`Any, Any` doesn't bring any new information.

Reply:
I was thinking about that and wondering if it should be `[str, Any]` but I'd be fine dropping it entirely.

Reply:
I've actually moved this to just `self.serialized_job: dict`, as I consider this to be `lazy initialized` in this context - no mypy warnings in this case.